db/migrate/20171028221157_add_reblogs_to_follows.rb

```diff
@@ -0,0 +1,21 @@
+require Rails.root.join('lib', 'mastodon', 'migration_helpers')
+
+class AddReblogsToFollows < ActiveRecord::Migration[5.1]
+  include Mastodon::MigrationHelpers
+
+  safety_assured do
```

Comment:
This wrapping should not be necessary

Reply:
Hm, right you are. I thought strong_migrations complained otherwise, guess not(? or at least not when I just tried it).